guide/migrating-4.md

```diff
@@ -134,13 +134,13 @@ Here is an example of chained route handlers defined using `app.route()`.
 app.route('/book')
   .get(function(req, res) {
     res.send('Get a random book');
-  })
+  });
```

Comment:
@dougwilson you're right. Seems that I've been over-keen on adding the semicolons. I'll fix and make another pull request.